qcelemental/tests/test_molecule.py

```diff
@@ -364,8 +364,6 @@ def test_get_fragment(group_fragments, orient):
         assert dimers[0].get_hash() == dimers[3].get_hash()
         assert dimers[1].get_hash() == dimers[4].get_hash()
         assert dimers[2].get_hash() == dimers[5].get_hash()
-    else:
-        assert 0
```

Comment:
these are a check that if you don't do one of the known/coded combos, everything explodes. not strictly necessary, but I don't think they hurt.

Reply:
True, but in our current testing was unreachable because the Pytest parameterization was either `True, True` or `False, False`. So either LGTM caught that or..... it was a fluke. I can add it back in though for safety and we just say "meh" to the LGTM warning here.